libraries/chain/chain_config.cpp

```diff
@@ -45,6 +45,14 @@ void chain_config_v1::validate() const {
                "max action return value size should be less than MAX_SIZE_OF_BYTE_ARRAYS" );
 }
 
+void chain_config_v2::validate() const {
+   chain_config_v1::validate();
+   EOS_ASSERT( 1 <= max_sync_call_depth, action_validate_exception,
+               "max sync call depth should be at least 1" );
+   EOS_ASSERT( max_sync_call_data_size <= MAX_SIZE_OF_BYTE_ARRAYS, action_validate_exception,
+               "max ssync call data size should be less than MAX_SIZE_OF_BYTE_ARRAYS" );
```

Comment:
"max sync"

Reply:
also it might be nicer to output the value of `MAX_SIZE_OF_BYTE_ARRAYS` here instead of just "MAX_SIZE_OF_BYTE_ARRAYS"

Reply:
I now see this is just copied how v1 also prints it as  `MAX_SIZE_OF_BYTE_ARRAYS`. Can keep it like this for consistency (or change both I guess).

Reply:
https://github.com/AntelopeIO/spring/pull/1257/commits/318d440e9a8279df2ca491af899d031fca5d1da5
I updated v1's too.